### ISSUE TITLE
Fix logging in UI

### ISSUE THREAD

**Issue by NikMidas** [Midas-Protocol/monorepo#1126]
https://discord.com/channels/930077813585899561/930157130454024192/1053008645429936138

![Image](https://user-images.githubusercontent.com/103433798/207934677-e8da6cae-1210-4227-a81c-44c8313146ed.png)

**Comment by smartdevcom** [comment]
I guess this is fixed by @carlomazzaferro here https://github.com/Midas-Protocol/monorepo/commit/e5749f7801935a883c5b237a2cbbeceb561067f1

**Comment by carlomazzaferro** [comment]
> I guess this is fixed by @carlomazzaferro here https://github.com/Midas-Protocol/monorepo/commit/e5749f7801935a883c5b237a2cbbeceb561067f1

No, my fix isn't a real fix. Turning off any info logs is definitely not a solution

**Comment by smartdevcom** [comment]
> > I guess this is fixed by @carlomazzaferro here [e5749f7](https://github.com/Midas-Protocol/monorepo/commit/e5749f7801935a883c5b237a2cbbeceb561067f1)
> 
> No, my fix isn't a real fix. Turning off any info logs is definitely not a solution

Do we need to display only errors?
Then we can set 'LogLevel.ERROR`

**Comment by carlomazzaferro** [comment]
> > > I guess this is fixed by @carlomazzaferro here [e5749f7](https://github.com/Midas-Protocol/monorepo/commit/e5749f7801935a883c5b237a2cbbeceb561067f1)
> > 
> > 
> > No, my fix isn't a real fix. Turning off any info logs is definitely not a solution
> 
> Do we need to display only errors? Then we can set 'LogLevel.ERROR` to see only errors and to disable warnings

No, we need to fix the `duplicate definitions ...` that show up in the logs, either with ethers 6 or otherwise

**Comment by NikMidas** [comment]
Dependent on another task to complete before this one can be reviewed again

**Comment by NikMidas** [comment]
@carlomazzaferro working on an item that will resolve this as well

**Comment by NikMidas** [comment]
Per @peetzweg this should be resolved by the next ether.js release

**Comment by peetzweg** [comment]
This line of code currently prevents this log from showing. Showing the log slows down the frontend significantly. 

We should remove this asap.

https://github.com/Midas-Protocol/monorepo/blob/effd8623ddd97d4b8ab7af0e933eec043bc8af8c/packages/sdk/src/MidasSdk/index.ts#L56


There are several of issues describing this in the ethersjs repo. One of them is this one. It advertises it will be solved in v6 of ethersjs.

https://github.com/ethers-io/ethers.js/issues/1029

Reducing the ABIs manually is not an option for us as of now.

**Comment by peetzweg** [comment]
Unblocked now. Ethers 6.0 is supposed to fix this. 
We should give it a try.

https://www.npmjs.com/package/ethers?activeTab=versions
